src/store/reducers/authUserReducer.ts

```diff
@@ -0,0 +1,74 @@
+interface IAuthedUser {
+  id: null | string;
+  userRole: string;
+  name: string;
+}
+
+interface ILoginState {
+  readonly authLoading: boolean;
+  readonly authedUser: IAuthedUser | null;
+  readonly loginError: string;
+}
+
+const LoginState: ILoginState = {
+  authLoading: false,
+  authedUser: null,
+  loginError: ''
+};
+
+export enum ActionsType {
```

Comment:
all interfaces and enums should be stored in separate file in order to be reusable